src/router.js

```diff
@@ -18,7 +35,9 @@ export const mountRoutes = (app, provider) => {
 
   app.use('/interaction', urlencoded({ extended: false }));
 
-  app.get('/', (req, res) => { res.sendStatus(200); });
+  app.get('/', (req, res) => {
```

Comment:
We should check with @rdubigny about code style before submitting automated formatting changes.

We might need a `.prettierrc` on this project, unless it is supposedly up to eslint to handle formatting ?